common/config/rush/command-line.json

```diff
@@ -68,11 +68,19 @@
     {
       "commandKind": "global",
       "name": "build:all-flavors",
-      "summary": "Build all flavor in one command (mainly for api update)",
-      "description": "Build all flavor in one command (mainly for api update). This command will switch current flavor to beta.",
+      "summary": "Build all packages for all flavors in one command",
+      "description": "Build all packages for all flavors in one command. This command will switch current flavor to beta.",
       "shellCommand": "(rush switch-flavor:stable && rush rebuild) || rush switch-flavor:beta && rush rebuild",
       "safeForSimultaneousRushProcesses": true
     },
+    {
+      "commandKind": "global",
+      "name": "build:npm-all-flavors",
```

Comment:
The name is a bit confusing.

`build:regen-api-all-flavors` ? It'll reflect the main use-case (and I'll be able to easily find it when I need it by running `rush --help`)